python/cudf/cudf/benchmarks/bench_cudf_io.py

```diff
@@ -82,3 +93,97 @@ def bench_json(benchmark, file_path, use_buffer, dtype):
         orient="records",
         dtype=dtype,
     )
+
+
+@pytest.mark.parametrize("dtype", datatype)
+@pytest.mark.parametrize("null_frequency", null_frequency)
+def bench_to_csv(benchmark, dtype, null_frequency, run_bench):
+    if not run_bench:
+        pytest.skip("Pytest variable run_bench not passed as True")
```

Comment:
Is this something that should happen at the beginning of every benchmark? 

If so, we should consider some way of writing it once and reusing everywhere. For example:

1. A pytest fixture that we can use for all benchmarks
2. A custom decorator we can decorate each benchmark with